components/lvgl_esp32_drivers/lvgl_spi_conf.h

```diff
@@ -0,0 +1,72 @@
+/**
+ * @file lvgl_spi_conf.h
+ *
+ */
+
+#ifndef LVGL_SPI_CONF_H
+#define LVGL_SPI_CONF_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/*********************
+ *      INCLUDES
+ *********************/
+
+/*********************
+ *      DEFINES
+ *********************/
+// DISPLAY PINS
+#define DISP_SPI_MOSI CONFIG_LVGL_DISP_SPI_MOSI
+#define DISP_SPI_CLK CONFIG_LVGL_DISP_SPI_CLK
+#define DISP_SPI_CS CONFIG_LVGL_DISP_SPI_CS
+
+// TOUCHPAD PINS
+#define TP_SPI_MOSI CONFIG_LVGL_TOUCH_SPI_MOSI
+#define TP_SPI_MISO CONFIG_LVGL_TOUCH_SPI_MISO
+#define TP_SPI_CLK  CONFIG_LVGL_TOUCH_SPI_CLK
+#define TP_SPI_CS   CONFIG_LVGL_TOUCH_SPI_CS
```

Comment:
This symbols are already defined in `lvgl_touch/touch_driver.h`, can we use those?

Reply:
I thought i removed those from`lvgl_touch/tp_spi.h`. I like that all pins are defined in one file (for instance if you don't want to use `menuconfig` any more).